meson.build

```diff
@@ -0,0 +1,179 @@
+project('pyvrp', 'cpp', default_options : ['cpp_std=c++20'])
+
+# The following lines define compiler and linker options. The options are set
+# per buildtype: for example, debug builds are also given a few coverage 
+# settings, whereas release builds optimise for speed.
+if get_option('buildtype') == 'release'
+    add_project_arguments('-O3', language: 'cpp')
+    add_project_link_arguments('-flto', language: 'cpp')
```

Comment:
You could set `-Db_lto=true` as an option to Meson, or add to the `project()` function the keyword argument:
```meson
project(..., default_options: ['b_lto=true'])
```

to make that option be the default without manually specifying it on the meson command line.